regression/contracts/quantifiers-loop-02/main.c

```diff
@@ -1,12 +1,14 @@
 #include <assert.h>
 
+const int MAX = 3;
```

Comment:
A macro would be preferable here, but perhaps this MAX is unnecessary.
```suggestion
#define MAX_ARRAY_SIZE 3
```

Reply:
fixed